core/src/main/java/org/apache/any23/extractor/yaml/ElementsProcessor.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2017 The Apache Software Foundation.
```

Comment:
No need to include Copyright in any new files. These quickly become out-of-date from year to year. We just put a generic license header and then in NOTICE include the Copyright year. Can you please remove the Copyright? Thanks @jgrzebyta